migrations/versions/60a681c463a2_create_donors_override.py

```diff
@@ -0,0 +1,33 @@
+"""create_donors_override
+
+Revision ID: 60a681c463a2
+Revises: 6fe71f4f1aba
```

Comment:
Mergnuli jsme PR s migrací a tato změna zajistí, že půjdou migrace za sebou a nerozvětví se.
```suggestion
Revises: 971ddb205e0c
```